lib/workers/repository/extract/manager-files.ts

```diff
@@ -1,18 +1,65 @@
 import is from '@sindresorhus/is';
-import type { WorkerExtractConfig } from '../../../config/types';
+import type {
+  CustomManager,
+  ManagerConfig,
+  MatchStringsStrategy,
+} from '../../../config/types';
 import { logger } from '../../../logger';
 import {
   extractAllPackageFiles,
   extractPackageFile,
   get,
 } from '../../../modules/manager';
-import type { PackageFile } from '../../../modules/manager/types';
+import type {
+  CustomExtractConfig,
+  ExtractConfig,
+  PackageFile,
+} from '../../../modules/manager/types';
 import { readLocalFile } from '../../../util/fs';
 
+export function getExtractConfig(config: ManagerConfig): ExtractConfig {
+  const result: ExtractConfig = {
+    npmrc: config.npmrc,
+    aliases: config.aliases as Record<string, string>, // temporary fix : not for prod
+    skipInstalls: config.skipInstalls as boolean, // temporary fix : not for prod
```

Comment:
```suggestion
    aliases: config.aliases!,
    skipInstalls: config.skipInstalls!,
```

is it for typescript?